src/Cmts.ml

```diff
@@ -41,7 +41,7 @@ end) : sig
 
   val children : t -> Itv.t -> Itv.t list
 
-  val dump : t -> Fmt.t
+  val dump : t -> Formatting.t
```

Comment:
The renaming is due to a name collision (Bos requires the Fmt package)

Reply:
I'm sorry, but no, this does not seem to be worth it. Is there something I'm overlooking? Bos should wrap its internal dependencies instead of polluting the global namespace.

Reply:
The problem is that the library named `fmt` (https://github.com/dbuenzli/fmt) is a dependency of `bos`

Reply:
I know, but that doesn't change things wrt this renaming. There are just a few small functions using bos, if it means not using bos and using the standard versions of them instead until bos is fixed to not pollute the global namespace with its internal deps, then so be it.